packages/database/schema.prisma

```diff
@@ -393,22 +393,24 @@ model Integration {
 }
 
 model Environment {
-  id                   String           @id @default(cuid())
-  createdAt            DateTime         @default(now()) @map(name: "created_at")
-  updatedAt            DateTime         @updatedAt @map(name: "updated_at")
-  type                 EnvironmentType
-  product              Product          @relation(fields: [productId], references: [id], onDelete: Cascade)
-  productId            String
-  widgetSetupCompleted Boolean          @default(false)
-  surveys              Survey[]
-  people               Person[]
-  actionClasses        ActionClass[]
-  attributeClasses     AttributeClass[]
-  apiKeys              ApiKey[]
-  webhooks             Webhook[]
-  tags                 Tag[]
-  segments             Segment[]
-  integration          Integration[]
+  id                    String           @id @default(cuid())
+  createdAt             DateTime         @default(now()) @map(name: "created_at")
+  updatedAt             DateTime         @updatedAt @map(name: "updated_at")
+  type                  EnvironmentType
+  product               Product          @relation(fields: [productId], references: [id], onDelete: Cascade)
+  productId             String
+  widgetSetupCompleted  Boolean          @default(false)
```

Comment:
since the data migration doesn't depend on the `widgetSetupCompled` I think we can safely delete this?

Reply:
we're actually using `widgetSetupCompleted` in the data migration, to only migrate the environments where there was any sync before
![image](https://github.com/formbricks/formbricks/assets/56182734/d6207236-9659-4246-bf6e-66480caf9604)